action/execution.go

```diff
@@ -78,6 +82,44 @@ func (ex *Execution) Data() []byte { return ex.data }
 // Payload is same as Data()
 func (ex *Execution) Payload() []byte { return ex.data }
 
+// AccessList returns the access list
+func (ex *Execution) AccessList() types.AccessList { return ex.accessList }
+
+func toAccessListProto(list types.AccessList) []*iotextypes.AccessTuple {
+	if len(list) == 0 {
+		return nil
+	}
+	proto := make([]*iotextypes.AccessTuple, len(list))
+	for i, v := range list {
+		proto[i] = &iotextypes.AccessTuple{}
+		proto[i].Address = hex.EncodeToString(v.Address.Bytes())
+		if numKey := len(v.StorageKeys); numKey > 0 {
+			proto[i].StorageKeys = make([]string, numKey)
```

Comment:
same here

Reply:
yeah, there is test case with `v.StorageKeys = nil`